gp-pay-per-word/gpppw-round-up-pricing.php

```
@@ -0,0 +1,210 @@
+	}
+
+	/**
+	 * Round up the calculated price to the nearest whole dollar
+	 *
+	 * @param float $price The calculated price
+	 * @param object $price_field The price field object
+	 * @param object $word_field The word field object  
+	 * @param int $word_count The calculated word count
+	 * @return float The rounded up price
+	 */
+	public function round_up_price( $price, $price_field, $word_field, $word_count ) {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$word_field'. (undefined)

(UnusedFormalParameter)